usecases/sanction_check_usecase.go

```diff
@@ -448,6 +452,10 @@ func (uc SanctionCheckUsecase) FilterOutWhitelistedMatches(ctx context.Context,
 	sanctionCheck.Matches = matchesAfterWhitelisting
 	sanctionCheck.Count = len(sanctionCheck.Matches)
 
+	if sanctionCheck.Count == 0 {
```

Comment:
I don't know if this was on purpose or an omission, but it seemed that if all matches were whitelisted, the status of the screening kept its iniital value.

It seems appropriate to change it to `no_hit` if we go down to zero match.